src/fragments/lib/graphqlapi/js/working-with-files.mdx

```diff
@@ -0,0 +1,984 @@
+The Storage and GraphQL API categories can be used together to associate a file, such as an image or video, with a particular record. For example, you might create a `Person` model with a profile picture, or a `Post` model with an associated image. With Amplify's GraphQL API and Storage categories, you can reference the file within the model itself to create an association.
+
+To get started, run the following in an existing Amplify project:
+
+```bash
+npm i @aws-amplify/ui-react  #For Authenticator component
+amplify add auth             #Select default configuration
+amplify add storage          #Select S3 with read/write access
+amplify add api              #Select Cognito User Pool for authorization type
+```
+
+When prompted, use the following schema:
+
+```graphql
+type Song @model {
```

Comment:
For copy and paste-ability, let's just make sure the global auth rule is set or `@auth(rules: [{ allow: public }])`